wasm2c/examples/build-standlone/uvwasi-rt-main.c

```diff
@@ -0,0 +1,76 @@
+/* Link this with wasm2c output and uvwasi runtime to build a standalone app */
+#include <stdio.h>
+#include <stdlib.h>
+#include "uvwasi.h"
+#include "uvwasi-rt.h"
+
+#define MODULE_NAME input
+#define MODULE_HEADER "input.h"
+#include MODULE_HEADER
+
+//force pre-processor expansion of m_name
+#define __module_init(m_name) Z_## m_name ##_init_module()
+#define module_init(m_name) __module_init(m_name)
```

Comment:
I also agree with @keithw that it would be nice to avoid these macros and assume that wasm2c was run with the correct `-m` flag.

Reply:
For sure, they have been removed in 
[9703be9 (https://github.com/WebAssembly/wabt/pull/2002/commits/9703be965ffd275d8e4cebda5e8f3c873ec7054f)